Makefile

```diff
@@ -1630,7 +1634,7 @@ ifneq ($(USER_MODULE_NAME),)
 MODULE_NAME := $(USER_MODULE_NAME)
 endif
 
-ifneq ($(KERNELRELEASE),)
```

Comment:
What's the reasoning for this change, especially removing the conditional?
I may recall things incorrectly, but I thought that setting `CONFIG_RTL8812AU_8821AU = m` caused issue.